crates/rolldown_common/src/module/normal_module.rs

```diff
@@ -37,6 +37,8 @@ pub struct NormalModule {
   pub default_export_ref: SymbolRef,
   pub sourcemap_chain: Vec<rolldown_sourcemap::SourceMap>,
   pub is_included: bool,
+  // The runtime module and module which path starts with `\0` shouldn't generate sourcemap. Ref see https://github.com/rollup/rollup/blob/master/src/Module.ts#L279.
+  pub sourcemap: bool,
```

Comment:
Let's add `is_virtual` method in `NormalModule` directly and use it. It makes eaiser to understand why this modules are excluded from generating sourcemap instead of wondering why `sourcemap` is `false` and who control this value.

Reply:
DOne